src/features/CourtCaseDetails/Sidebar/PncDetails/PncDetails.styles.tsx

```diff
@@ -22,23 +22,48 @@ const CourtCase = styled.div`
   font-size: var(--default-font-size);
 `
 
-const CourtCaseHeader = styled.div`
+const CourtCaseHeaderContainer = styled.div`
   display: flex;
-  flex-direction: column;
+  justify-content: space-between;
 
-  padding: 15px 20px;
   background-color: ${gdsLightGrey};
   border-bottom: solid 1px ${gdsMidGrey};
+
+  &:hover {
+    background-color: #dfdfe0;
+
+    .chevron {
+      color: white;
+      background: black;
+    }
+  }
+`
+
+const ChevronContainer = styled.div`
+  display: flex;
+  justify-content: flex-end;
+  flex-grow: inherit;
+  align-items: center;
+  margin-right: 15px;
+`
+
+const CourtCaseHeader = styled.div`
+  width: 100%;
+
+  margin: 15px 20px;
```

Comment:
Can we please create a separate file for the `PncCourtCaseAccordion` styled components?

Reply:
Created a separate styled component